diff -u -r curl-orig/curl.spec curl-patched/curl.spec
--- curl-orig/curl.spec	2017-03-07 16:26:44.000000000 +0100
+++ curl-patched/curl.spec	2018-03-09 11:30:23.667154612 +0100
@@ -433,13 +433,6 @@
 
 make %{?_smp_mflags}
 
-%check
-export LD_LIBRARY_PATH=$RPM_BUILD_ROOT%{_libdir}
-cd tests
-make %{?_smp_mflags}
-# use different port range for 32bit and 64bit build, thus make it possible
-# to run both in parallel on the same machine
-./runtests.pl -a -b%{?__isa_bits}90 -p -v
 
 %install
 rm -rf $RPM_BUILD_ROOT
