diff -u -r original/davix.spec modified/davix.spec
--- original/davix.spec	2020-04-29 14:09:47.000000000 +0200
+++ modified/davix.spec	2021-07-29 14:27:24.305798955 +0200
@@ -1,5 +1,6 @@
 # unversionned doc dir F20 change https://fedoraproject.org/wiki/Changes/UnversionedDocdirs
 %{!?_pkgdocdir: %global _pkgdocdir %{_docdir}/%{name}-%{version}}
+%global cmake /usr/bin/cmake
 
 Name:				davix
 Version:			0.7.6
@@ -24,11 +25,6 @@
 # davix-copy dependencies
 BuildRequires:                  gsoap-devel
 
-# unit tests and abi check
-%if %{?fedora}%{!?fedora:0} >= 10 || %{?rhel}%{!?rhel:0} >= 6
-BuildRequires:                  abi-compliance-checker
-%endif
-
 Requires:                       %{name}-libs%{?_isa} = %{version}-%{release}
 
 
@@ -87,7 +83,7 @@
 %cmake \
 -DDOC_INSTALL_DIR=%{_pkgdocdir} \
 -DENABLE_THIRD_PARTY_COPY=TRUE \
--DENABLE_HTML_DOCS=TRUE \
+-DCMAKE_INSTALL_PREFIX=/usr/ \
 .
 make %{?_smp_mflags}
 make doc
@@ -131,7 +127,6 @@
 %files doc
 %{_pkgdocdir}/LICENSE
 %{_pkgdocdir}/RELEASE-NOTES.md
-%{_pkgdocdir}/html/
 
 %files tests
 %{_bindir}/davix-unit-tests
