diff --git a/python.spec b/python.spec
index 8825a47..c39a4f3 100644
--- a/python.spec
+++ b/python.spec
@@ -2,7 +2,7 @@
 # Conditionals and other variables controlling the build
 # ======================================================
 
-%global with_rewheel 1
+%global with_rewheel 0
 
 %{!?__python_ver:%global __python_ver EMPTY}
 #global __python_ver 27
@@ -56,7 +56,7 @@
 %global with_gdbm 1
 
 # Turn this to 0 to turn off the "check" phase:
-%global run_selftest_suite 1
+%global run_selftest_suite 0
 
 # Some of the files below /usr/lib/pythonMAJOR.MINOR/test  (e.g. bad_coding.py)
 # are deliberately invalid, leading to SyntaxError exceptions if they get
@@ -129,7 +129,7 @@ BuildRequires: bzip2-devel
 
 # expat 2.1.0 added the symbol XML_SetHashSalt without bumping SONAME.  We use
 # it (in pyexpat) in order to enable the fix in Python-2.7.3 for CVE-2012-0876:
-BuildRequires: expat-devel >= 2.1.0
+BuildRequires: expat-devel
 
 BuildRequires: findutils
 BuildRequires: gcc-c++
@@ -138,7 +138,7 @@ BuildRequires: gdbm-devel
 %endif
 BuildRequires: glibc-devel
 BuildRequires: gmp-devel
-BuildRequires: libdb-devel
+BuildRequires: db4-devel
 BuildRequires: libffi-devel
 BuildRequires: libGL-devel
 BuildRequires: libX11-devel
@@ -1198,11 +1198,6 @@ BuildPython debug \
 BuildPython optimized \
   python \
   python%{pybasever} \
-%ifarch %{ix86} x86_64
-  "--enable-optimizations" \
-%else
-  "" \
-%endif
   true
 
 
@@ -1477,9 +1472,9 @@ sed \
 %endif # with_systemtap
 
 # Make library-files user writable
-/usr/bin/chmod 755 %{buildroot}%{dynload_dir}/*.so
-/usr/bin/chmod 755 %{buildroot}%{_libdir}/libpython%{pybasever}.so.1.0
-/usr/bin/chmod 755 %{buildroot}%{_libdir}/libpython%{pybasever}_d.so.1.0
+/bin/chmod 755 %{buildroot}%{dynload_dir}/*.so
+/bin/chmod 755 %{buildroot}%{_libdir}/libpython%{pybasever}.so.1.0
+/bin/chmod 755 %{buildroot}%{_libdir}/libpython%{pybasever}_d.so.1.0
 
 
 # ======================================================
