diff -r -u original/xrootd/bindings/python/setup.py.in patched/xrootd/bindings/python/setup.py.in
--- original/xrootd/bindings/python/setup.py.in 2020-02-03 08:55:35.000000000 +0000
+++ patched/xrootd/bindings/python/setup.py.in  2020-02-21 13:48:15.466745239 +0000
@@ -24,7 +24,7 @@
 cfg_vars["CXX"] = " ".join(cxxl)
 cfg_vars["PY_CXXFLAGS"] = "${CMAKE_CXX_FLAGS}"

-cfg_vars['LDSHARED'] = cfg_vars['LDSHARED'] + " -Wl,-rpath,${XRDCL_INSTALL}"
+#cfg_vars['LDSHARED'] = cfg_vars['LDSHARED'] + " -Wl,-rpath,${XRDCL_INSTALL}"

 sources = list()
 depends = list()
diff -r -u original/xrootd.spec patched/xrootd.spec
--- original/xrootd.spec        2020-02-03 09:55:51.000000000 +0000
+++ patched/xrootd.spec 2020-02-21 13:47:58.377652275 +0000
@@ -77,11 +77,6 @@
 BuildRequires: json-c-devel

 BuildRequires: python2-devel
-%if %{?fedora}%{!?fedora:0} >= 13
-BuildRequires: python3-devel
-%else
-BuildRequires: python%{python3_pkgversion}-devel
-%endif

 BuildRequires: openssl-devel

@@ -300,21 +295,6 @@
 %description -n python2-xrootd
 Python 2 bindings for XRootD

-%if %{?_with_python3:1}%{!?_with_python3:0}
-#-------------------------------------------------------------------------------
-# python3
-#-------------------------------------------------------------------------------
-%package -n python3-%{name}
-Summary:       Python 3 bindings for XRootD
-Group:         Development/Libraries
-%if %{?fedora}%{!?fedora:0} >= 13
-%{?python_provide:%python_provide python3-%{name}}
-%endif
-Requires:      %{name}-client-libs%{?_isa} = %{epoch}:%{version}-%{release}
-
-%description -n python3-xrootd
-Python 3 bindings for XRootD
-%endif

 #-------------------------------------------------------------------------------
 # doc
@@ -464,12 +444,6 @@
 popd
 %endif

-%if %{?_with_python3:1}%{!?_with_python3:0}
-# build python3 bindings
-pushd build/bindings/python
-%py3_build
-popd
-%endif

 #-------------------------------------------------------------------------------
 # Installation
@@ -586,12 +560,6 @@
 install -m 644 -p packaging/common/xrootd.pp \
     %{buildroot}%{_datadir}/selinux/packages/%{name}/%{name}.pp

-%if %{?_with_python3:1}%{!?_with_python3:0}
-# install python3 bindings
-pushd build/bindings/python
-%py3_install
-popd
-%endif

 %clean
 rm -rf $RPM_BUILD_ROOT
@@ -883,11 +851,6 @@
 %files -n python2-%{name} -f xrootd/build/PYTHON_INSTALLED_FILES
 %defattr(-,root,root,-)

-%if %{?_with_python3:1}%{!?_with_python3:0}
-%files -n python3-%{name}
-%defattr(-,root,root,-)
-%{python3_sitearch}/*
-%endif

 %files doc
 %defattr(-,root,root,-)
