diff -r -u original/xrootd.spec patched/xrootd.spec
--- original/xrootd.spec	2019-05-13 14:37:34.000000000 +0200
+++ patched/xrootd.spec	2019-05-21 11:51:38.789782992 +0200
@@ -77,12 +77,6 @@
 BuildRequires: json-c-devel

 BuildRequires: python2-devel
-%if %{?fedora}%{!?fedora:0} >= 13
-BuildRequires: python3-devel
-%else
-BuildRequires: python%{python3_pkgversion}-devel
-%endif
-
 BuildRequires: openssl-devel

 BuildRequires: selinux-policy-devel
@@ -300,22 +294,6 @@
 %description -n python2-xrootd
 Python 2 bindings for XRootD

-%if %{?_with_python3:1}%{!?_with_python3:0}
-#-------------------------------------------------------------------------------
-# python3
-#-------------------------------------------------------------------------------
-%package -n python3-%{name}
-Summary:       Python 3 bindings for XRootD
-Group:         Development/Libraries
-%if %{?fedora}%{!?fedora:0} >= 13
-%{?python_provide:%python_provide python3-%{name}}
-%endif
-Requires:      %{name}-client-libs%{?_isa} = %{epoch}:%{version}-%{release}
-
-%description -n python3-xrootd
-Python 3 bindings for XRootD
-%endif
-
 #-------------------------------------------------------------------------------
 # doc
 #-------------------------------------------------------------------------------
@@ -463,13 +441,6 @@
 popd
 %endif

-%if %{?_with_python3:1}%{!?_with_python3:0}
-# build python3 bindings
-pushd build/bindings/python
-%py3_build
-popd
-%endif
-
 #-------------------------------------------------------------------------------
 # Installation
 #-------------------------------------------------------------------------------
@@ -585,13 +556,6 @@
 install -m 644 -p packaging/common/xrootd.pp \
     %{buildroot}%{_datadir}/selinux/packages/%{name}/%{name}.pp

-%if %{?_with_python3:1}%{!?_with_python3:0}
-# install python3 bindings
-pushd build/bindings/python
-%py3_install
-popd
-%endif
-
 %clean
 rm -rf $RPM_BUILD_ROOT

@@ -879,12 +843,6 @@
 %files -n python2-%{name} -f xrootd/build/PYTHON_INSTALLED_FILES
 %defattr(-,root,root,-)

-%if %{?_with_python3:1}%{!?_with_python3:0}
-%files -n python3-%{name}
-%defattr(-,root,root,-)
-%{python3_sitearch}/*
-%endif
-
 %files doc
 %defattr(-,root,root,-)
 %doc %{_docdir}/%{name}-%{version}
